docs/installation/presto-cli-installation.rst

```diff
@@ -9,18 +9,17 @@ The Presto CLI provides a terminal-based interactive shell for running queries.
 Download the `presto-cli` and copy it to the location you want to run it from. This location may be any node that has network access to the coordinator. Then rename it to presto and make it executable with chmod +x:
 ::
 
- $ mv presto-cli-0.115-executable.jar presto
+ $ mv presto-cli-*-executable.jar presto
```

Comment:
What do you think of mv presto-cli-<PrestoVersion>-executable.jar presto?

That way they can't copy paste, but will be forced to substitute in the presto version.